### PR TITLE
Add unit test coverage for operator blocks

### PR DESCRIPTION
**Pull request by thisandagain**
Implements minimal test coverage for the `scratch3_operators` block module. Resolves GH-103
## Before

image
image
## After

image
image

### CODE REVIEW

File: test/unit/blocks_operators.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+test('multiply', function (t) {
+    t.strictEqual(blocks.multiply({NUM1:'2', NUM2:'2'}), 4);
+    t.strictEqual(blocks.multiply({NUM1:'foo', NUM2:'bar'}), 0);
+    t.end();
+});
+
+test('divide', function (t) {
+    t.strictEqual(blocks.divide({NUM1:'2', NUM2:'2'}), 1);
+    t.strictEqual(blocks.divide({NUM1:'1', NUM2:'0'}), Infinity);   // @todo
```
Comment (thisandagain) [inline test/unit/blocks_operators.js:31]: Will file issue to discuss this. Follow-up from:
https://github.com/LLK/scratch-flash/issues/1060

File: README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 #### Scratch VM is a library for representing, running, and maintaining the state of computer programs written using [Scratch Blocks](https://github.com/LLK/scratch-blocks).
 
 [![Build Status](https://travis-ci.org/LLK/scratch-vm.svg?branch=develop)](https://travis-ci.org/LLK/scratch-vm)
+[![Coverage Status](https://coveralls.io/repos/github/LLK/scratch-vm/badge.svg?branch=develop)](https://coveralls.io/github/LLK/scratch-vm?branch=develop)
```
Comment (tmickel) [inline README.md:5]: 😎

File: test/unit/blocks_operators.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+test('not', function (t) {
+    t.strictEqual(blocks.not({OPERAND:true}), false);
+    t.strictEqual(blocks.not({OPERAND:false}), true);
+    t.end();
+});
+
+test('random', function (t) {
+    var min = 0;
+    var max = 100;
+    var result = blocks.random({FROM:0, TO:1});
```
Comment (tmickel) [inline test/unit/blocks_operators.js]: I think you wanted `blocks.random{FROM:min, TO:max});` here

**Comment by tmickel** [comment]
Very cool that it worked out to be so easy to add tests for these, and again great to have these to verify things will be executing as we expect. Made one tiny comment, but otherwise looks good to me!

File: test/unit/blocks_operators.js (hunk 3 above)
Comment (thisandagain) [reply test/unit/blocks_operators.js]: Good catch.

**Pull request merged**